src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesArray = sourceString.split(';');
+  const stylesObject = {};
+
+  stylesArray.forEach((style) => {
+    const [property, value] = style.split(':').map((item) => item.trim());
```

Comment:
This line is doing two operations: splitting the string and trimming the results. The chaining is well done, but consider handling the case where a style declaration might not contain a colon, which would cause an undefined value in the resulting array.